docs/tutorial.rst

```diff
@@ -2,197 +2,277 @@ Tutorial
 ========
 
 UVData: File conversion
----------
+-----------------------
 Converting between tested data formats
 
 a) miriad (aipy) -> uvfits
 **************************
 ::
 
-  from pyuvdata import UVData
-  UV = UVData()
-  miriad_file = 'pyuvdata/data/new.uvA'
-  UV.read_miriad(miriad_file)  # this miriad file is known to be a drift scan
-  UV.write_uvfits('new.uvfits', force_phase=True, spoof_nonessential=True)  # write out the uvfits file
+  >>> from pyuvdata import UVData
+  >>> UV = UVData()
+
+  This miriad file is known to be a drift scan
+  >>> miriad_file = 'pyuvdata/data/new.uvA'
+  >>> UV.read_miriad(miriad_file)
+
+  Write out the uvfits file
+
+  >>> UV.write_uvfits('tutorial.uvfits', force_phase=True, spoof_nonessential=True)
+  The data are in drift mode and do not have a defined phase center. Phasing to zenith of the first timestamp.
```

Comment:
Yes, this output is what the doctest checks to see that it got and it fails otherwise. I don't think I can format it differently, but I might be able to format the comments differently.